osu.Game/Overlays/OnlineOverlay.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Overlays
+{
+    public abstract class OnlineOverlay<T> : FullscreenOverlay<T>
```

Comment:
there is nothing inherently online about this overlay, it doesn't do anything with API or the sort. actually closer to a scrolling overlay if anything.

Reply:
It was "WebOverlay" at first, ppy's request was to change it to `OnlineOverlay`.
It's used only to represent web pages though, even if there's api isn't involved (it will be in the future with https://github.com/ppy/osu/pull/10774 placeholders in mind, which should be handled here probably, but not sure yet)

Reply:
this could use `OnlineViewContainer` on `content`, yeah? that way the online overlay will actually do online stuff (I was also wondering why that mentioned PR doesn't use `OnlineViewContainer` at all, not even a word)

Reply:
The naming I suggested was based on the usage of the classs (all current and future usages are going to be "online" overlays, as in client side implementations of the web site). I'm open to an alternative if something better is available.

Reply:
Meh, going back to this can't say I have anything better to suggest. The name fits for what the class is supposed to be.